elements/tests.bst

```diff
@@ -0,0 +1,14 @@
+kind: stack
+
+depends:
+- tests/sdk/basin.bst
+- tests/sdk-build-depends/hotdoc-modular-framework.bst
+- tests/sdk-depends/emeus.bst
+- tests/sdk-depends/eos-knowledge-content-renderer.bst
+- tests/sdk-depends/eos-shard.bst
+- tests/sdk-depends/maxwell.bst
+- tests/sdk-depends/xapian-glib.bst
+# - tests/sdk/eos-knowledge-lib.bst
+# - tests/sdk/eos-metrics.bst
```

Comment:
Are these comment intended?

Reply:
Oh, yeah. I commented those out since they error out at the moment, but shouldn't really be like that :)

The problem with eos-metrics is related to the test environment and resolved in this pull request: https://github.com/endlessm/eos-metrics/pull/42. For eos-knowledge-lib, there's something weird going on which is probably outside the scope of this branch, and once that's fixed tests.bst should run happily!